pygments/lexers/configs.py

```diff
@@ -70,6 +66,71 @@ def analyse_text(text):
         return text[0] == '[' and text[npos-1] == ']'
```

Comment:
This will still catch the other files if this analyzer happens to run before them, so it will need to return 0.99 or something smaller than 1 now.